packages/frontend/core/src/pages/workspace/detail-page/detail-page.tsx

```diff
@@ -67,16 +61,8 @@ interface DetailPageLayoutProps {
   sidebar: ReactNode;
 }
 
-// disable animation to avoid UI flash
-function useEnableAnimation() {
-  const [enable, setEnable] = useState(false);
-  useEffect(() => {
-    window.setTimeout(() => {
-      setEnable(true);
-    }, 500);
-  }, []);
-  return enable;
-}
+const MIN_SIDEBAR_WIDTH = 320;
```

Comment:
https://github.com/toeverything/AFFiNE/blob/87cb68f7fb456644ffd949a7dc62693597969dbb/packages/frontend/core/src/pages/workspace/detail-page/editor-sidebar/editor-sidebar.css.ts#L9

need change default width to 320